### PR TITLE
fix vulnerabilities and webpack version

### PR DESCRIPTION
**Pull request by KavyaReddy732**
changed the versions of babel-loader,dotenv-webpack,webpack,webpack-cli and the devdependencies 
   "case-sensitive-paths-webpack-plugin": "^2.3.0",
    "concurrently": "^7.0.0",
    "install-peers": "^1.0.3",
    "nodemon": "^2.0.15",
    "sequelize-cli": "^6.4.1",
    "webpack-dev-server": "^4.7.4"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,14 +48,16 @@
...
-    "case-sensitive-paths-webpack-plugin": "^2.3.0"
+    "case-sensitive-paths-webpack-plugin": "^2.3.0",
+    "concurrently": "^7.0.0",
+    "install-peers": "^1.0.3",
+    "nodemon": "^2.0.15",
+    "sequelize-cli": "^6.4.1",
+    "webpack-dev-server": "^4.7.4"
   }
 }
+
```
Comment (orhantoy) [inline package.json]: I think we can remove this empty line.

**Review by orhantoy — APPROVED** [review]
Looks good 🎉 
How did you test the changes?

**Comment by KavyaReddy732** [comment]
Tested using the existing DB fo my mealsharing app. i made some changes in webpack.config as the new version of package doesn't allow some of the modules.

**Comment by KavyaReddy732** [comment]
Screenshot 2022-03-30 at 16 09 12
Screenshot 2022-03-30 at 16 10 02

**Comment by orhantoy** [comment]
Thanks for fixing this @KavyaReddy732 🎉

**Pull request merged**